### ISSUE TITLE
Links inside a VSCode Walkthrough description are not displayed in VS Code accessible view

### ISSUE THREAD

**Issue by Achyuth-ibm-z** [microsoft/vscode#247149]
When opening the VS Code accessible view while on a walkthrough, the hyperlinks do not get rendered to the view. For example, in the attached image, a hyperlink that should say "Install Z Open Editor" with a link to the documentation should appear, but instead there is an empty space.
 
This is true for all walkthrough links. 

How to replicate:
1. Open VS Code and turn on screen reader accessibility mode
2. Open any walkthrough step that contains links
3. Open the accessible view with "option + f2" on mac or "alt + f2" on Windows
4. Notice how the text is missing where the hyperlink should be

<img width="1401" alt="Image" src="https://github.com/user-attachments/assets/660f5376-cf7d-4411-a69e-df2e0aca92cf" />
